### PR TITLE
expanded useCart hook

### PR DESCRIPTION
**Pull request by MickhaelSh**
expanded useCart hook

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/orders/cart/filled-cart/filled-cart.js [hunk 1]
```diff
@@ -195,7 +196,7 @@ const FilledCart = ({ items, cartOperations }) => {
                   placeholder={t('cart.promoPlaceHolder')}
                   variant={TEXT_FIELD_VARIANT.OUTLINED}
                   inputRef={certificateAndPromoInput}
-                  disabled={disable}
+                  disabled={!!(certificateData || promoCode)}
```
Comment (Pipuss) [inline src/containers/orders/cart/filled-cart/filled-cart.js:199]: Use Boolean() instead !!

File: src/containers/orders/order/your-order/your-order.js [hunk 2]
```diff
@@ -49,8 +49,8 @@ const YourOrder = ({ ...props }) => {
       </Typography>
       <Divider variant='fullWidth' />
       <List className={styles.yourOrderList} data-testid='orderList'>
-        {cart
-          ? cart.map((item) => (
+        {cartItems
+          ? cartItems.map((item) => (
```
Comment (Pipuss) [inline src/containers/orders/order/your-order/your-order.js:53]: Bad practice using arr.map() inside jsx

**Review by Pipuss — CHANGES_REQUESTED** [review]

File: src/containers/orders/cart/filled-cart/filled-cart.js (hunk 1 above)
Comment (MickhaelSh) [reply src/containers/orders/cart/filled-cart/filled-cart.js]: fixed

File: src/containers/orders/order/your-order/your-order.js (hunk 2 above)
Comment (MickhaelSh) [reply src/containers/orders/order/your-order/your-order.js]: fixed

**Review by Pipuss — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**